pkgs/tools/package-management/clib/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, curl  }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.2";
+  name = "clib-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/clibs/clib/archive/${version}.tar.gz";
```

Comment:
That looks like a generated archive, better use fetchzip/fetchFromGithub so that the hash is calculated over the extracted files (instead of the archive itself).